src/averageAges.js

```diff
@@ -20,6 +20,14 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const MenCollection = people.filter(person => person.sex === 'm'
```

Comment:
```suggestion
  const Men = people.filter(person => person.sex === 'm'
```
also you can create helper function to find out gender of a person